docs/rules/no-implicit-coercion.md

```diff
@@ -106,6 +107,40 @@ var s = String(foo);
 foo = String(foo);
 ```
 
+### disallowTemplateString
```

Comment:
It's now `"disallowTemplateString"` in the docs, but `"disallowTemplateShorthand"` in the code.